docs/docs/recipes.md

```diff
@@ -922,6 +922,98 @@ export default ({ pageContext: { allPokemon } }) => (
 - More on using unstructured data in [Using Gatsby without GraphQL](/docs/using-gatsby-without-graphql/)
 - When and how to [query data with GraphQL](/docs/querying-with-graphql/) for more complex Gatsby sites
 
+### Sourcing content from Drupal
+
+#### Prerequisites
+
+- A [Gatsby site](/docs/quick-start)
+- A [Drupal](http://drupal.org) site
+- The [JSON:API module](https://www.drupal.org/project/jsonapi) installed and enabled on the Drupal site
+
+#### Directions
+
+1. Install the `gatsby-source-drupal` plugin.
+
+```
+npm install --save gatsby-source-drupal
+```
+
+2. Edit your `gatsby-config.js` file to enable the plugin and configure it.
+
+```javascript:title=gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-source-drupal`,
+      options: {
+        baseUrl: `https://your-website/`,
+        apiBase: `api`, // optional, defaults to `jsonapi`
+      },
+    },
+  ],
+}
+```
+
```

Comment:
I went back and forth on this a bit... do we want all recipes like Drupal, etc. to include examples of both static GraphQL calls as well as generating pages using `createPages`? One thought I had is that users probably want to quickly get to generating pages from their WP, Drupal CMS quickly. On the other hand, once the user has a grasp of GraphQL and the sourcing of data, theoretically the process of dynamically creating pages should be agnostic.

Another thing to consider is if the docs have dozens of places that all show examples of using `createPages` then it can be messy to keep them updated in the future.

Reply:
Great points. Showing a template of some kind is useful so people can get something up and running quickly with a standalone format - and I think this accomplishes that. In the interest of keeping things moving, let's keep it as-is and we can iterate on it in the future if needed.